core/domain/exp_jobs_one_off.py

```diff
@@ -57,6 +60,32 @@ def map(exploration_model):
     def reduce(exp_id, list_of_exps):
         pass
 
+class ExpSummariesContributorsOneOffJob(jobs.BaseMapReduceJobManager):
+    """ One-off job that finds the user id's of the contributors
+    (defined as any human who has made a positive (i.e. not just
+    a reversion) for each exploration.
+    """
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationSnapshotMetadataModel]
+
+    @staticmethod
+    def map(item):
+        _VERSION_DELIMITER = '-'
```

Comment:
Okay re:top of file. 

I'm turning in for the night pretty quick, but two things:
1) Now I'm getting the following error: "'property' object has no attribute 'rfind'". Is there some method I'm missing to get the value of the property in the model? Been googling and couldn't come up with much.
2) This method won't cover Kerry's use case-- she's using ExplorationRightsSnapshotContentModel, which doesn't inherit from BaseSnapshotMetadataModel. Does that at all change where you think I should put it?

Thanks again!